diff --git a/node_modules/@ethersproject/contracts/lib.esm/index.js b/node_modules/@ethersproject/contracts/lib.esm/index.js
index bd5f668..bb7fec8 100644
--- a/node_modules/@ethersproject/contracts/lib.esm/index.js
+++ b/node_modules/@ethersproject/contracts/lib.esm/index.js
@@ -231,8 +231,7 @@ function buildEstimate(contract, fragment) {
         });
     };
 }
-function addContractWait(contract, tx) {
-    const wait = tx.wait.bind(tx);
+const addContractWait = (contract, tx) => {
     tx.wait = (confirmations) => {
         return wait(confirmations).then((receipt) => {
             receipt.events = receipt.logs.map((log) => {
