diff --git a/node_modules/@thorswap-lib/midgard-sdk/lib/index.js b/node_modules/@thorswap-lib/midgard-sdk/lib/index.js
index 20c8b43..d91c53b 100644
--- a/node_modules/@thorswap-lib/midgard-sdk/lib/index.js
+++ b/node_modules/@thorswap-lib/midgard-sdk/lib/index.js
@@ -2227,8 +2227,8 @@ var Configuration = /** @class */ (function () {
 var MIDGARD_STAGENET_URL = 'https://stagenet-midgard.ninerealms.com';
 var MIDGARD_TESTNET_URL = 'https://testnet.midgard.thorchain.info';
 var midgardServiceUrls = {
-    thorchain: 'https://midgard.thorchain.info',
-    thorswap: 'https://midgard.thorswap.net',
+    thorchain: 'https://midgard.ninerealms.com',
+    thorswap: 'https://midgard.ninerealms.com',
     ninerealms: 'https://midgard.ninerealms.com',
 };
 
