diff --git a/node_modules/node-fetch/lib/index.es.js b/node_modules/node-fetch/lib/index.es.js
index 79d717b..beb0568 100644
--- a/node_modules/node-fetch/lib/index.es.js
+++ b/node_modules/node-fetch/lib/index.es.js
@@ -656,7 +656,7 @@ function writeToStream(dest, instance) {
 }
 
 // expose Promise
-Body.Promise = global.Promise;
+Body.Promise = window.Promise;
 
 /**
  * headers.js
diff --git a/node_modules/node-fetch/lib/index.js b/node_modules/node-fetch/lib/index.js
index 337d6e5..bd8f175 100644
--- a/node_modules/node-fetch/lib/index.js
+++ b/node_modules/node-fetch/lib/index.js
@@ -660,7 +660,7 @@ function writeToStream(dest, instance) {
 }
 
 // expose Promise
-Body.Promise = global.Promise;
+Body.Promise = window.Promise;
 
 /**
  * headers.js
diff --git a/node_modules/node-fetch/lib/index.mjs b/node_modules/node-fetch/lib/index.mjs
index ace669f..65e2eea 100644
--- a/node_modules/node-fetch/lib/index.mjs
+++ b/node_modules/node-fetch/lib/index.mjs
@@ -654,7 +654,7 @@ function writeToStream(dest, instance) {
 }
 
 // expose Promise
-Body.Promise = global.Promise;
+Body.Promise = window.Promise;
 
 /**
  * headers.js
