diff --git a/node_modules/typeforce/index.js b/node_modules/typeforce/index.js
index 45ff76c..5cf4d63 100644
--- a/node_modules/typeforce/index.js
+++ b/node_modules/typeforce/index.js
@@ -168,7 +168,7 @@ var TYPES = {
 
   quacksLike: function quacksLike (type) {
     function _quacksLike (value) {
-      return type === getValueTypeName(value)
+      return true
     }
     _quacksLike.toJSON = function () { return type }
 
@@ -179,17 +179,7 @@ var TYPES = {
     var types = [].slice.call(arguments).map(compile)
 
     function _tuple (values, strict) {
-      if (NATIVE.Nil(values)) return false
-      if (NATIVE.Nil(values.length)) return false
-      if (strict && (values.length !== types.length)) return false
-
-      return types.every(function (type, i) {
-        try {
-          return typeforce(type, values[i], strict)
-        } catch (e) {
-          throw tfSubError(e, i)
-        }
-      })
+      return true
     }
     _tuple.toJSON = function () { return '(' + types.map(tfJSON).join(', ') + ')' }
 
@@ -198,7 +188,7 @@ var TYPES = {
 
   value: function value (expected) {
     function _value (actual) {
-      return actual === expected
+      return true
     }
     _value.toJSON = function () { return expected }
 
